README.rst

```diff
@@ -4,109 +4,114 @@ NVIDIA DALI
-portability of training and inference workflows, and code
-maintainability.
-
-NVIDIA DALI (R), NVIDIA's Data Loading Library, is a collection of highly
-optimized building blocks, and an execution engine, to accelerate the
-pre-processing of the input data for deep learning applications. DALI
-provides both the performance and the flexibility to accelerate
-different data pipelines as one library. This library can
-then be easily integrated into different deep learning training and
-inference applications.
+The NVIDIA Data Loading Library (DALI) is a library for data loading and
+pre-processing to accelerate deep learning applications. It provides a
+collection of highly optimized building blocks for loading and processing
+image, video and audio data. It can be used as a portable drop-in replacement
+for built in data loaders and data iterators in popular deep learning frameworks.
+
+Deep learning applications require complex, multi-stage data processing pipelines
+that include loading, decoding, cropping, resizing, and many other augmentations.
+These data processing pipelines, which are currently executed on the CPU, have become a 
+bottleneck, limiting the performance and scalability of training and inference.
+
+DALI addresses the problem of the CPU bottleneck by offloading data preprocessing to the
+GPU. Additionally, DALI relies on its own execution engine, built to maximize the throughput
+of the input pipeline. Features such as prefetching, parallel execution, and batch processing
+are handled transparently for the user.
+
+In addition, the deep learning frameworks have multiple data pre-processing implementations,
+resulting in challenges such as portability of training and inference workflows, and code
+maintainability. Data processing pipelines implemented using DALI are portable because they
+can easily be retargeted to TensorFlow, PyTorch, MXNet and PaddlePaddle. 
+
+.. image:: /dali.png
+    :width: 800
+    :align: center 
+    :alt: DALI Diagram
 
 Highlights
 ----------
-
-- Full data pipeline--accelerated from reading the disk to getting
-  ready for training and inference.
-- Flexibility through configurable graphs and custom operators.
-- Support for image classification and segmentation workloads.
-- Ease of integration through direct framework plugins and open
-  source bindings.
-- Portable training workflows with multiple input formats: JPEG,
-  PNG (fallback to CPU), TIFF (fallback to CPU), BMP (fallback to CPU),
-  raw formats, LMDB, RecordIO, TFRecord.
-- Extensible for user-specific needs through open source license.
+- Easy-to-use functional style Python API.
+- Multiple data formats support - LMDB, RecordIO, TFRecord, COCO, JPEG, JPEG 2000, WAV, FLAC, OGG, H.264, VP9 and HEVC.
+- Portable accross popular deep learning frameworks: TensorFlow, PyTorch, MXNet, PaddlePaddle.
+- Supports CPU and GPU execution.
+- Scalable across multiple GPUs.
+- Flexible graphs lets developers create custom pipelines.
```

Comment:
_graphs let_ or _graph lets_

Also it's not clear for me what does it mean. What makes our graphs flexible?

Reply:
I copied it from the developer page. It means that users can write their own processing graphs.